tests/dashboard/test_permissions.py

```diff
@@ -988,3 +988,33 @@ def test_staff_can_remove_user(staff_client, staff_user, permission_manage_users
 
     response = staff_client.get(url)
     assert response.status_code == 200
+
+
+def test_payment_index_view_cannot_be_accessed_without_permision(
```

Comment:
Personally I see here 4 tests. `test_payment_index_view_cannot_be_accessed_without_permision` can be converted to:
```
test_payment_index_view_cannot_be_accessed_without_permision()
    url = reverse("dashboard:payments-index")
     assert not staff_user.has_perm("extensions.manage_plugins")
     response = staff_client.get(url)
     assert response.status_code == 302
```
```
test_payment_index_view_can_be_accessed_with_permision()
     url = reverse("dashboard:payments-index")
     staff_user.user_permissions.add(permission_manage_plugins)
     response = staff_client.get(url)
     assert response.status_code == 200
```

`test_only_staff_user_with_proper_permission_can_update_payment_gateway` can be converted to 
```
test_only_staff_user_cannot_update_payment_gateway_without_permission()
    url = reverse("dashboard:configure-payment", args=["Braintree"])
     assert not staff_user.has_perm("extensions.manage_plugins")
     response = staff_client.get(url)
     assert response.status_code == 302
```
```
test_only_staff_user_with_proper_permission_can_update_payment_gateway():

      staff_user.user_permissions.add(permission_manage_plugins)
     response = staff_client.get(url)
     assert response.status_code == 200
```